File: app/src/main/java/com/zzzmode/appopsx/ui/main/group/IPermGroupView.java
package com.zzzmode.appopsx.ui.main.group;

import com.zzzmode.appopsx.ui.model.PermissionGroup;
import java.util.List;

/**
 * Created by zl on 2017/7/17.
 */
interface IPermGroupView {
  void changeTitle(int groupPosition,int childPosition,boolean allowed);
  void refreshItem(int groupPosition,int childPosition);
  void showList(List<PermissionGroup> value);
  void showError(Throwable e);
}

File: app/src/main/kotlin/com/zzzmode/appopsx/ui/main/group/IPermGroupView.kt
package com.zzzmode.appopsx.ui.main.group

import com.zzzmode.appopsx.ui.model.PermissionGroup

/**
 * Created by zl on 2017/7/17.
 */
internal interface IPermGroupView {
    fun changeTitle(groupPosition: Int, childPosition: Int, allowed: Boolean)
    fun refreshItem(groupPosition: Int, childPosition: Int)
    fun showList(value: List<PermissionGroup>)
    fun showError(e: Throwable)
}
